lib/tezos_client/tools/convert_to_hash/timestamp.rb

```diff
@@ -8,6 +8,8 @@ def decode
           if data.key? :int
             Time.zone.at(data[:int].to_i)
           elsif data.key? :string
+            return Time.zone.at(data[:string].to_i) if data[:string].match?(/\A[\d]*\z/)
```

Comment:
chippotage: pour éviter que `""` passe la regex
```suggestion
            return Time.zone.at(data[:string].to_i) if data[:string].match?(/\A[\d]+\z/)
```

Reply:
Chipotage v2 :

```suggestion
            return Time.zone.at(data[:string].to_i) if data[:string].match?(/\A\d*\z/)
```

Reply:
Merge des chipotages :

```suggestion
            return Time.zone.at(data[:string].to_i) if data[:string].match?(/\A\d+\z/)
```